apps/fluent-tester/src/E2E/Avatar/pages/AvatarPageObject.ts

```diff
@@ -1,7 +1,39 @@
-import { JSAVATAR_TESTPAGE, HOMEPAGE_AVATAR_BUTTON } from '../../../FluentTester/TestComponents/Avatar/consts';
+import {
+  JSAVATAR_TESTPAGE,
+  HOMEPAGE_AVATAR_BUTTON,
+  AVATAR_TEST_COMPONENT,
+  AVATAR_SECONDARY_TEST_COMPONENT,
+} from '../../../FluentTester/TestComponents/Avatar/consts';
 import { BasePage, By } from '../../common/BasePage.win';
 
+export const enum AvatarComponentSelector {
+  PrimaryComponent, //this._primaryComponent
+  SecondaryComponent, //this._secondaryComponent
+}
 class AvatarPageObject extends BasePage {
+  getPrimaryComponentAttribute(attribute: string): string {
+    return this._primaryComponent.getAttribute(attribute);
+  }
+  getSecondaryComponentAttribute(attribute: string): string {
+    return this._secondaryComponent.getAttribute(attribute);
+  }
+  getAvatarAccessibilityLabel(componentSelector: AvatarComponentSelector): string {
+    return componentSelector == AvatarComponentSelector.SecondaryComponent
+      ? this._secondaryComponent.getAttribute('Name')
+      : this._primaryComponent.getAttribute('Name');
+  }
+
+  getAvatarAccessibilityHint(componentSelector: AvatarComponentSelector): string {
+    return componentSelector == AvatarComponentSelector.SecondaryComponent
+      ? this._secondaryComponent.getAttribute('HelpText')
+      : this._primaryComponent.getAttribute('HelpText');
+  }
+
+  getAvatarAccessibilityRole(componentSelector: AvatarComponentSelector): string {
+    return componentSelector == AvatarComponentSelector.SecondaryComponent
+      ? this._secondaryComponent.getAttribute('ControlType')
+      : this._primaryComponent.getAttribute('ControlType');
+  }
```

Comment:
I added these methods siumilarly to what we have in other tests. But I'm not sure if it's good to create a new method for each attribute. So I added getAttribute methods for each component

Reply:
That seems okay to me. It would be nice for all component page objects to do the same but that can be addressed in another PR. @samuelfreiberg, was there any particular reason for having individual methods for each attribute?

Reply:
Yeah, I was thinking about that. Because  we have the same methods in all the components